extras/nginx/Makefile.j2

```diff
@@ -9,7 +9,8 @@ KEY_PATH={{ KEY_PATH }}/{{ KEY_FILE }}
 KEY_SIZE={{ KEY_SIZE }}
 SERVER_URL={{ SERVER_URL }}
 SITES_AVAILABLE_DIR={{ SITES_AVAILABLE_DIR }}
-SITES_ENABLED_DIR= {{ SITES_ENABLED_DIR }}
+SITES_ENABLED_DIR={{ SITES_ENABLED_DIR }}
+SERVER_CFG_DIR={{ SITES_ENABLED_DIR }}/..
```

Comment:
I'm just wondering if this will bite us somewhere.. we could always define {{SERVER_CFG_DIR}} and then `{{SERVER_CFG_DIR}}/sites-enabled` (more reliable)..